draft-ietf-jsonpath-base.md

```diff
@@ -1411,25 +1411,25 @@ a non-empty nodelist. The table also lists the (immediate) subtypes of each type
 
 | Type                  | Abstract Instances                       | Subtypes                                   |
 | :--                   | :----------------                        | :------                                    |
-| `OptionalNodeOrValue` | `Node(n)`, `Value(v)`, `Nothing`         | `OptionalNode`, `OptionalValue`            |
-| `OptionalNode`        | `Node(n)`, `Nothing`                     |                                            |
-| `OptionalValue`       | `Value(v)`, `Nothing`                    | `OptionalNode`, `Value`, `OptionalBoolean` |
-| `Value`               | `Value(v)`                               | `Boolean`                                  |
-| `OptionalBoolean`     | `Value(true)`, `Value(false)`, `Nothing` | `Boolean`                                  |
-| `Boolean`             | `Value(true)`, `Value(false)`            |                                            |
-| `OptionalNodes`       | `Nodes(nl)`, `Nothing`                   | `OptionalNode`                             |
+| `OptionalNodeTypeOrValueType` | `Node(n)`, `Value(v)`, `Nothing`         | `OptionalNodeType`, `OptionalValueType`            |
```

Comment:
I wonder if the double "type" is necessary as it doesn't conform to the pattern of adding "Type" to the end of each type name.
```suggestion
| `OptionalNodeOrValueType` | `Node(n)`, `Value(v)`, `Nothing`         | `OptionalNodeType`, `OptionalValueType`            |
```

Reply:
Yeah, that's probably an artifact of replacing strings.  I'll fix.

Reply:
✔️